julia/heat/New_heat.jl

```diff
@@ -0,0 +1,55 @@
+using FastBroadcast
+using BenchmarkTools
```

Comment:
Can you please add the license to this file?

https://github.com/diehlpk/async_heat_equation/blob/df43c66d0eec780748b92f1232ec4746d7a53ad7/julia/heat/heat_ghosts.jl#L1